src/handle_action.cpp

```diff
@@ -1401,6 +1401,9 @@ static void loot()
     menu.query();
     flags = ( menu.ret >= 0 ) ? menu.ret : None;
     bool recache = false;
+    // clear fetch history previous multiple activity.
+    player_character.fetch_history.clear();
```

Comment:
The fetch history should belong to NPC instead of player character.

Reply:
fetch history now belong to NPC. In the future these should belong to player_activity but it is needed to clean up codes , because multi_activity is not supposed to resume.